### ISSUE TITLE
NTR: Acyl-CoA oxidase low in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8197]
New term request Acyl-CoA oxidase low in blood (segal_070726212906):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000217
- label: Reduced circulating acyl-CoA oxidase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:2894756
- def: The activity of acyl-CoA oxidase activity in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8197
- synonyms:
